pyproject.toml

```diff
@@ -11,21 +11,21 @@ classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
     "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3 :: Only",
     "Topic :: Security :: Cryptography",
     "Topic :: Text Processing :: Markup :: XML",
     "Typing :: Typed"
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
 cryptography = ">=2.8"
-lxml = ">=4.4.1"
+lxml = "^5.2"
```

Comment:
is there a reason we need 5.2 (as opposed to say 'any 5.x')?

Reply:
Yeah not really, let me change to 5.x 🙏

Reply:
@ojii Changed to `^5.0`